gdk/common/exceptions/InitError.py

```diff
@@ -0,0 +1,37 @@
+class InitException(Exception):
+    def __init__(self, reason="", solution="", exception=None, help_text=""):
+        err_details = ""
+        if exception:
+            err_details = f"Error details: {exception}\n"
+        self.message = f"{reason}\n{err_details}{solution}{help_text}"
+        super().__init__(self.message)
```

Comment:
This should be generic GDK cli base exception like`GdkException`. Move to a common generic errors file.

If you still want to keep `InitException` for init command specific exception handling during `InitCommand` execution, then you can create a new class `InitException(GdkException)`.

Reply:
Right, worked on those exact changes but this will also go in build command exception. Just wanted to introduce this change in next PR to keep it simple.